less/project/filter.less

```diff
@@ -0,0 +1,27 @@
+// Filter mixin
+// Pass in single or multi arguments to apply filter effects
+// Can only be used in CSS
+// --------------------------------------------------
```

Comment:
Move filter 'Examples' to sit within 'Filter mixin' comment for consistency with other mixin comments? Lines 9-12 can then be removed.